### DISCUSSION TITLE
Ideas for more autocompleting snippets in latex.hsnips

### DISCUSSION THREAD

**Discussion by universemaster** [sleepymalc/VSCode-LaTeX-Inkscape#13] [Ideas]
I found some more ideas for autocompleting snippets for your latex.hsnips at https://github.com/joeyuping/quick_latex_obsidian detailed in the [README](https://github.com/joeyuping/quick_latex_obsidian/blob/master/README.md).

Unfortunately, these ideas have been implemented in [typescript](https://github.com/joeyuping/quick_latex_obsidian/blob/master/main.ts) - but I strongly suspect all functionality could be implemented in hsnips directly.

**Comment by sleepymalc** [comment]
Ah I somehow miss the mail about this discussion thread, Sry! Which snippet do you want? I think all of them are quite easy to implement and some of them are already included.

**Comment by sydee281** [comment]
I've been using [Obsidian Latex Suite](https://github.com/artisticat1/obsidian-latex-suite), which is inspired by Quick Latex for Obsidian. The specific features I think would be really useful are
- Auto-fraction that includes greek letters.  Ex. `\pi r/` -> `\frac{\pi r}{}`
- Auto enlarge brackets that contain `\frac`, `\sum`, or `\prod` 
- "Matrix shortcuts" that let you tab through matrix, array, align, or cases environments
